### ISSUE TITLE
Add support for ELECTRON_BUILDER_BINARIES_CUSTOM_DIR

### ISSUE THREAD

**Issue by mirayashi** [develar/app-builder#99]
Currently it is impossible to customize the directory part of the URL when downloading `[winCodeSign-version] .7z` for example

This line needs to be changed so we can specify a custom value from env instead of `id`:

https://github.com/develar/app-builder/blob/0bc38fb5e76c38ab24e36acf7a17887370e31fb4/pkg/download/tool.go#L63

The main electron-builder repo already supports `ELECTRON_BUILDER_BINARIES_CUSTOM_DIR` so why not use it here?

**Comment by timothyjmtan** [comment]
I agree, this will help teams who are trying to get it working while behind a corporate firewall which presents loads of restrictions. 

I have an offline copy of `winCodeSign-2.6.0.7z`, but I'm not able to point app-builder-bin towards this because it's hardcoded to download from github which will always fail.

Can we please incorporate this as a quick fix? Ideally it should check for the presence of files in `ELECTRON_BUILDER_BINARIES_CUSTOM_DIR`, before attempting to download.

**Comment by timothyjmtan** [comment]
I've also tried the Docker image, but the build process still attempts to download winCodeSign. If all dependencies (including winCodeSign, etc) could be packaged into the docker image, that would be very helpful and helps circumvent the above connectivity issues. Otherwise, teams will be totally blocked from using this tool.

https://hub.docker.com/r/electronuserland/builder
